workflows/align_gisaid/config.yaml

```diff
@@ -0,0 +1,10 @@
+configfile:
+  - defaults/parameters.yaml
+
+# NOTE: when the number of cpus is modified, it would be prudent to modify workflows/transform_gisaid/transform.sh.
+cores: 48
```

Comment:
damnnnnnn that's a lot of cores

Reply:
Well, we could probably do with fewer cores, but there aren't machines with 384GB of memory with fewer cores.

Reply:
ok so it seems 384GB of memory isn't enough to consistently finish the alignment.  I increased the memory requirement so now we are using r5.16xlarge (64 cores, 512GB RAM), but decreased the cores because it's not really that helpful beyond a certain point.  hopefully we can schedule other things onto those machines.